src/app/core/components/Navbar/NavbarGlobalSearch.tsx

```diff
@@ -96,6 +96,13 @@ const Navbar = (props: NavbarProps) => {
     }
   }, [filters]);
 
+  useEffect(() => {
```

Comment:
nit: you can extract this to a function called resetGlobalSearch or something like that.